### ISSUE TITLE
Accessibility migration from react native web 18 to 19 is broken

### ISSUE THREAD

**Issue by ayroblu** [necolas/react-native-web#2665]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

I've been doing the migration from React Native 18 to 19 and noticed a couple of bugs

accessibilityLabel for Image doesn't fill the alt text, have to use aria-label
role=label for View doesn't work, have to use accessibilityRole

### Expected behavior

Backwards compatibility, forwards compatibility

### Steps to reproduce

Migrate to rnw19

### Test case

https://codesandbox.io/p/sandbox/proud-cache-dyz8ff?file=%2Fsrc%2FApp.js

**Comment by ayroblu** [comment]
@necolas ?
